### PR TITLE
Query and update

### PR DESCRIPTION
**Pull request by KatIsCoding**
In this update more data is added to the database, a new property `species` is added to the schema and more queries using `group by` and aggregators are added

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -7,3 +7,12 @@ SELECT date_of_birth from animals where name = 'Agumon' or name = 'Pickachu';
 SELECT name, escape_attempts from animals where weight_kg > 10.5;
 SELECT * from animals where neutered = true;
 SELECT * from animals where weight_kg between 10.4 and 17.3;
+
+/* Second Excercise Queries */
+
```
Comment (oxenprogrammer) [inline queries.sql:12]: **Changes Required**
You're missing some queries
- [x] UPDATE the animals table by setting the species column to unspecified, then roll back the change.
- [x] `UPDATE` the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- [x] `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set.
- [x] Commit the transaction.
- [x] Inside a transaction delete all records in the animals table, then roll back the transaction.
- [x] `DELETE` all animals born after Jan 1st, 2022
- [x] Create a savepoint for the transaction
- [x] `UPDATE` all animals' weight to be their weight multiplied by -1.
- [x] * Rollback to the savepoint
- [x] `UPDATE` all animals' weights that are negative to be their weight multiplied by -1.
- [x] Commit transaction

File: schema.sql [hunk 2]
```diff
@@ -9,5 +9,6 @@ CREATE TABLE animals (
     escape_attempts integer NOT NULL,
     neutered boolean NOT NULL,
     weight_kg decimal NOT NULL,
+    species text,
```
Comment (oxenprogrammer) [inline schema.sql]: **Changes Required**
- [x] We shouldn't add the `species` column directly to the `animal` table but rather use the `ALTER` command to do so. Let us try and make these changes.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @KatIsCoding, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾🎉✔**
Hello @KatIsCoding 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**